pakyow-support/lib/pakyow/support/dir_walk.rb

```diff
@@ -0,0 +1,49 @@
+module Pakyow
+  module Support
+    module WalkDir
+      DOT = ".".freeze
+
+      refine Dir.singleton_class do
```

Comment:
If we just did `refine Dir` it would add `walk` as an instance method. But we want it to be available on the class as `Dir.walk`.